base/applications/calc/lang/bg-BG.rc

```diff
@@ -135,17 +129,17 @@ BEGIN
     PUSHBUTTON "Дроб", IDC_BUTTON_F, 286, 140, 24, 18, BS_CENTER | BS_VCENTER |
                BS_NOTIFY | BS_OWNERDRAW | WS_TABSTOP
     GROUPBOX "", IDC_STATIC, 147, 14, 163, 20
-    CONTROL "ЧДума", IDC_RADIO_QWORD, "Button", BS_AUTORADIOBUTTON |
+    CONTROL "Qword", IDC_RADIO_QWORD, "Button", BS_AUTORADIOBUTTON |
```

Comment:
You're un-translating here.

Reply:
Because # 1 I put files without extension ".rc".
So, # 2 I revert all files.
And # 3 are the final files with extension ".rc".

Reply:
Please, look inside the last commit.